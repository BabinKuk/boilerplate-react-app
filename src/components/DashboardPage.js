import React from 'react';

const DashboardPage = () => (
    <div>
        DashboardPage content. Add components here
    </div>
);

export default DashboardPage;