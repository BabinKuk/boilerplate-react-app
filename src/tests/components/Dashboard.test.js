import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../components/DashboardPage';

test('Test Dashboard Page rendering', () => {
    const wrapper = shallow(<DashboardPage />);
    expect(wrapper).toMatchSnapshot();
});
