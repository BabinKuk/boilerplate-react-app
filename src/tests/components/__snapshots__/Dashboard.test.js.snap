// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Dashboard Page rendering 1`] = `
<div>
  DashboardPage content. Add components here
</div>
`;
